src/porepy/compositional/base.py

```diff
@@ -153,11 +161,11 @@ class Compound(Component):
     the :attr:`~Component.fraction` of the compound, i.e. the moles/mass of them are
     given by a product of mixture density, compound fraction and tracer fraction.
     :attr:`tracer_fraction_of` are assigned by the AD interface
```

Comment:
What does "simplified, but meaningfully generalized," mean (start of docstring)?

Reply:
Many fluid models out there like brine models or the black-oil model, treat multiple chemical species which are in reality all separate species, bundled into an artificial component (`Compound`).
For example H2O-NaCl brine is often treated as a such a bundled component, thought strictly speaking one should have H2O and two ion species. But the simplifications suffices for some target applications.

Reply:
Please add the gist of this explanation to the docstring.

Reply:
Thanks. I am fine with resolving.